webcompat/form.py

```diff
@@ -147,6 +148,12 @@ def domain_name(url):
     # testing if it's an http URL
     if url.startswith(SCHEMES):
         domain = urlparse.urlparse(url).netloc
+        if domain.lower().startswith('www.'):
+            domain = domain.split('www.')[1]
+        port_number = re.search(r':\d+$', domain)
+        if port_number is not None:
+            if domain.endswith(port_number.group()):
+                domain = domain.split(port_number.group())[0]
     else:
```

Comment:
All of that said, I don't think we should normalize `www.` and `:port_number`. There are part of the useful information of a domain name and even clearly identify different web properties.